packages/codemod/src/codemods/replace-generatetext-text-property.ts

```diff
@@ -0,0 +1,46 @@
+import { createTransformer } from './lib/create-transformer';
+
+export default createTransformer((fileInfo, api, options, context) => {
+  const { j, root } = context;
+
+  // Find member expressions that match: result.text, result1.text, result2.text, etc.
+  // where result is an identifier (not already a member expression)
+  root
+    .find(j.MemberExpression)
+    .filter(path => {
+      const node = path.node;
+
+      // Must be accessing a property called 'text'
+      if (!j.Identifier.check(node.property) || node.property.name !== 'text') {
+        return false;
+      }
+
+      // The object must be a simple identifier (not a member expression)
+      if (!j.Identifier.check(node.object)) {
+        return false;
+      }
+
+      // The identifier should be a common generateText result variable name
+      const commonNames = [
```

Comment:
This is wrong approach and replaces all the places where result.text is used, for example using fetch with response variable then doing response.text() or any other result.text. I would like work on this to fix, how do I test my changes? I can see input/output files, want to know the cmd to test it